package.json

```diff
@@ -41,12 +43,17 @@
   "dependencies": {
     "@emotion/react": "^11.9.3",
     "@lukeed/uuid": "^2.0.0",
+    "@types/uuid": "^8.3.4",
+    "cheminfo-types": "^1.1.0",
+    "filelist-utils": "^0.5.0",
     "immer": "^9.0.15",
+    "jcampconverter": "^9.0.2",
     "lodash": "^4.17.21",
     "react-dropzone": "^14.2.1",
     "react-shadow": "^19.0.3",
     "tinycolor2": "^1.4.2",
-    "use-resize-observer": "^9.0.2"
+    "use-resize-observer": "^9.0.2",
+    "uuid": "^8.3.2"
```

Comment:
Please remove this dependency (and `@types/uuid`) and use `@lukeed/uuid` that is much smaller, faster and already used in the project.